systest/backup/nfs-backup/backup_test.go

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/worker"
+	"github.com/dgraph-io/dgraph/x"
+
+	"github.com/dgraph-io/dgo/v210"
+	"github.com/dgraph-io/dgo/v210/protos/api"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+var (
+	copyBackupDir  = "./data/backups_copy"
+	backupDir      = "./data/backup"
+	restoreDir     = "./data/restore"
+	testDirs       = []string{backupDir, restoreDir}
+	backupDstHA    = "/ha_backup"
+	backupDstNonHA = "/non_ha_backup"
+)
+
+func TestBackupHAClust(t *testing.T) {
```

Comment:
This test fails for `arm` machine. Is that expected?

Reply:
No, I am figuring out why its fail on arm

Reply:
@anurags92 ,I have set enviroment varibale to set NFS server image version on basis of host machine architecture.